server/src/internalClusterTest/java/org/opensearch/action/admin/indices/create/CreateIndexIT.java

```diff
@@ -304,6 +307,7 @@ public void onFailure(Exception e) {
             }
         }
         latch.await();
+        assertNull(deleteFailure.get());
```

Comment:
Do we want to provide a log of the error message for better debugging if the test fails with delete failure?

Reply:
It will log the error with a message like "Expected null but got <the result of toString() on the exception>"